core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -69,7 +69,11 @@ object ApiVersion {
     "0.11.0-IV1" -> KAFKA_0_11_0_IV1,
     // Introduced leader epoch fetches to the replica fetcher via KIP-101
     "0.11.0-IV2" -> KAFKA_0_11_0_IV2,
-    "0.11.0" -> KAFKA_0_11_0_IV2
+    "0.11.0" -> KAFKA_0_11_0_IV2,
+    // Introduced LeaderAndIsrRequest V1, UpdateMetadataRequest V4 and MetadataRequest V5 via KIP-112
+    // Introduced ProduceRequest V4 and FetchRequest V6 in order to use KafkaStorageException for KIP-112
```

Comment:
Thanks. I see. I have removed both ProduceRequest, FetchRequest and MetadataRequest from this comment.